ios/browser/api/ledger/brave_ledger.h

```diff
@@ -7,8 +7,8 @@
 #define BRAVE_IOS_BROWSER_API_LEDGER_BRAVE_LEDGER_H_
 
 #import <Foundation/Foundation.h>
-#import "ledger.mojom.objc.h"
-#import "ledger_types.mojom.objc.h"
+#import "core.mojom.objc.h"
+#import "core_types.mojom.objc.h"
```

Comment:
I feel like this _may_ have problems in the future with name duplication, we aren't planning on changing the ads one to match right? May need to adjust how we determine the name of the header generated somehow since we can't have duped names since all headers are copied to 1 framework on iOS

Reply:
Good point, and I was unsure about that. If I change the mojo file names to "rewards_core.mojom" and "rewards_core_types.mojom", will that work?

Reply:
Yeah right now we use the name of the mojom file to determine the generated header name (matches the Chromium mojom logic) so that would help namespace it better for us :) 

If we need I can always adjust that logic to prefix it in some way or something